include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -605,13 +644,12 @@ class MGTransferGlobalCoarsening : public dealii::MGTransferBase<VectorType>
   /**
    * Collection of the two-level transfer operators.
    */
-  const MGLevelObject<MGTwoLevelTransfer<dim, VectorType>> &transfer;
+  MGLevelObject<MGTwoLevelTransfer<dim, VectorType>> transfer;
```

Comment:
Since we need to modify the two-level operators, we need to create a copy here.

Reply:
Does this imply a difference in the user code?

Reply:
As it is now, one could initialize the two-level operators afterwards:

https://github.com/dealii/dealii/blob/a5d36df8c40be68774a6c51f68041b33e0515311/include/deal.II/multigrid/mg_transfer_global_coarsening.h#L630-L631